src/Sylius/Bundle/PayumBundle/Controller/PayumController.php

```diff
@@ -43,7 +42,6 @@ public function __construct(
         private OrderRepositoryInterface $orderRepository,
         private MetadataInterface $orderMetadata,
         private RequestConfigurationFactoryInterface $requestConfigurationFactory,
-        private ViewHandlerInterface $viewHandler,
```

Comment:
Removed as controllers are excluded from BC and there's no previous deprecation triggering.